Source/inv.cpp

```diff
@@ -75,14 +75,14 @@ const InvXY InvRect[73] = {
 	{ 538, 336 }, // inv row 4
 	{ 567, 336 }, // inv row 4
 	{ 596, 336 }, // inv row 4
-	{ 205, 385 }, // belt
-	{ 234, 385 }, // belt
-	{ 263, 385 }, // belt
-	{ 292, 385 }, // belt
-	{ 321, 385 }, // belt
-	{ 350, 385 }, // belt
-	{ 379, 385 }, // belt
-	{ 408, 385 }  // belt
+	{ 205+WIDTH_DIFF_2, 385+HEIGHT_DIFF }, // belt
+	{ 234+WIDTH_DIFF_2, 385+HEIGHT_DIFF }, // belt
+	{ 263+WIDTH_DIFF_2, 385+HEIGHT_DIFF }, // belt
+	{ 292+WIDTH_DIFF_2, 385+HEIGHT_DIFF }, // belt
+	{ 321+WIDTH_DIFF_2, 385+HEIGHT_DIFF }, // belt
+	{ 350+WIDTH_DIFF_2, 385+HEIGHT_DIFF }, // belt
+	{ 379+WIDTH_DIFF_2, 385+HEIGHT_DIFF }, // belt
+	{ 408+WIDTH_DIFF_2, 385+HEIGHT_DIFF }  // belt

```

Comment:
This causes the game to crash for me:
```
AddressSanitizer:DEADLYSIGNAL
=================================================================
==17337==ERROR: AddressSanitizer: SEGV on unknown address 0x6320004e77c9 (pc 0x55be305151e2 bp 0x7fffcaaeb110 sp 0x7fffcaaeb0c0 T0)
==17337==The signal is caused by a READ memory access.
    #0 0x55be305151e1 in dvl::DrawPanelBox(int, int, int, int, int, int) /home/ajenbo/code/diablo/devilutionX/Source/control.cpp:498
    #1 0x55be305192ad in dvl::DrawCtrlPan() /home/ajenbo/code/diablo/devilutionX/Source/control.cpp:757
    #2 0x55be30475f22 in dvl::DrawAndBlit() /home/ajenbo/code/diablo/devilutionX/Source/scrollrt.cpp:1111
    #3 0x55be2fed5bc9 in dvl::run_game_loop(unsigned int) /home/ajenbo/code/diablo/devilutionX/Source/diablo.cpp:138
    #4 0x55be2fed5a5d in dvl::StartGame(int, int) /home/ajenbo/code/diablo/devilutionX/Source/diablo.cpp:113
    #5 0x55be30101c6d in dvl::mainmenu_init_menu(int) /home/ajenbo/code/diablo/devilutionX/Source/mainmenu.cpp:145
    #6 0x55be30101c31 in dvl::mainmenu_single_player() /home/ajenbo/code/diablo/devilutionX/Source/mainmenu.cpp:133
    #7 0x55be30101ae9 in dvl::mainmenu_loop() /home/ajenbo/code/diablo/devilutionX/Source/mainmenu.cpp:102
    #8 0x55be2fed605f in dvl::WinMain(void*, void*, char*, int) /home/ajenbo/code/diablo/devilutionX/Source/diablo.cpp:274
    #9 0x55be2fe4eeec in main /home/ajenbo/code/diablo/devilutionX/SourceX/main.cpp:28
    #10 0x7fa266c7db6a in __libc_start_main (/lib/x86_64-linux-gnu/libc.so.6+0x26b6a)
    #11 0x55be2fd7e639 in _start (/home/ajenbo/code/diablo/devilutionX/build/devilutionx+0xd39639)
```
With out the the belt looks correct but the spell button is in the wron place.

Reply:
ah right forgot the spell button :p